### PR TITLE
Updated streams-best-practices.md

### PR DESCRIPTION
**Pull request by joyceseng**
Hi Ji Won
I made some direct edits and left several comments. please let me know if anything doesn't make sense. /Joyce

### CODE REVIEW

File: Content_Portal/add-organize-data/store-data/streams-best-practices.md [hunk 1]
```diff
@@ -1,75 +1,77 @@
...
+
+  <!-- I removed the following  and added a note in step 5 below telling them to leave it blank. --> <!-- 3. Use the stream name to provide a more readable name for the string than the stream ID itself. -->
+
+- Use the stream description field, metadata, and tags to capture any other relevant information about the stream. <!--Not essential, since this bullet has sub bullets, but if possible, I suggest the following paragraph is in an indented paragraph, under the bullet -->This information is useful for searching for specific streams in the system, especially as systems become large. If possible, use consistent patterns in the description, metadata, and tags.
+   - <!--These 3 following are sub-bullets, right? -->Use metadata for information that follows a specific or consistent pattern, such as the location, manufacturer, and site. 
+   - Use tags for simple information about the stream that does not adhere to any particular pattern.
+   - Use the description field for longer descriptions of the stream and what it represents.
+
+## Reading large volumes of data from streams by using API calls
```
Comment (GiselleLeeOSI) [inline Content_Portal/add-organize-data/store-data/streams-best-practices.md:21]:
```suggestion
## Reading large volumes of data from streams with API

```

File: Content_Portal/add-organize-data/store-data/streams-best-practices.md [hunk 2]
```diff
@@ -1,75 +1,77 @@
...
+
+  <!-- I removed the following  and added a note in step 5 below telling them to leave it blank. --> <!-- 3. Use the stream name to provide a more readable name for the string than the stream ID itself. -->
+
+- Use the stream description field, metadata, and tags to capture any other relevant information about the stream. <!--Not essential, since this bullet has sub bullets, but if possible, I suggest the following paragraph is in an indented paragraph, under the bullet -->This information is useful for searching for specific streams in the system, especially as systems become large. If possible, use consistent patterns in the description, metadata, and tags.
+   - <!--These 3 following are sub-bullets, right? -->Use metadata for information that follows a specific or consistent pattern, such as the location, manufacturer, and site.
```
Comment (GiselleLeeOSI) [inline Content_Portal/add-organize-data/store-data/streams-best-practices.md:17]: Just bringing it into your awareness that lines up to 20 are Laureen's content.

**Review by GiselleLeeOSI — COMMENTED** [review]
Joyce, thank you for your feedback. Looks like you had some comments added for the main level section. You might want to add Laureen as your reviewer since she drafted the content between lines 4 and 20.

File: Content_Portal/add-organize-data/store-data/streams-best-practices.md (hunk 2 above)
Comment (joyceseng) [reply Content_Portal/add-organize-data/store-data/streams-best-practices.md:17]: Thanks Ji Won! I had fogotten that. Is it ok if I just add Laureen as a reviewer of this PR??

Comment (GiselleLeeOSI) [reply Content_Portal/add-organize-data/store-data/streams-best-practices.md:17]: Of course! That sounds good, Joyce.

**Comment by GiselleLeeOSI** [comment]
@joyceseng , @lasato-OSI  I spoke with Chris and he is okay with having the information in the developer hub section. So I'll close out my PR [https://github.com/osisoft/OCS-Docs/pull/370](url).

**Comment by joyceseng** [comment]
Oh, thank you, Ji Won! I'm very glad that this topic led me to get to know the API functionality better, and to start considering how we should introduce it to "portal" users.

**Comment by joyceseng** [comment]
Closing this PR, as after discussion, the work took a different direction.

**Pull request closed without merging**